resolwe/flow/static/flow/processSchema.json

```diff
@@ -160,6 +160,33 @@
                             "type": "boolean"
                         }
                     }
+                },
+                "relations": {
+                    "description": "List of relations used by the process",
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "type": {
+                                "type": "string"
```

Comment:
@mstajdohar How about changing this type to `"enum": ["compare", "group", "background"]` to make it even more narrow?